packages/runtime-common/realm.ts

```diff
@@ -691,14 +703,17 @@ export class Realm {
           }
         }
 
-        let isWrite = ['PUT', 'PATCH', 'POST', 'DELETE'].includes(
-          request.method,
-        );
-        await this.checkPermission(
-          request,
-          requestContext,
-          isWrite ? 'write' : 'read',
-        );
+        let requiredPermission: 'read' | 'write' | 'realm-owner';
+        if (['_permissions'].includes(this.paths.local(new URL(request.url)))) {
```

Comment:
i'm using an array here because I anticipate we'll have future endpoints that will need `realm-owner` permissions.